.github/workflows/deploy.yml

```diff
@@ -26,11 +463,54 @@ jobs:
       - name: Get current app version
         run: echo "STAGING_VERSION=$(npm run print-version --silent)" >> "$GITHUB_ENV"
 
-      - name: 🚀 Create prerelease to trigger staging deploy 🚀
+      - name: Download all workflow run artifacts
```

Comment:
This is new. Looking carefully to try and validate that we setup up the paths correctly according to the docs: https://github.com/actions/download-artifact?tab=readme-ov-file#download-all-artifacts